ebml_matroska.xml

```diff
@@ -1340,4 +1340,25 @@ If this Element is used, then any TagLanguage Elements used in the same SimpleTa
     <documentation lang="en" purpose="definition">The values of the Tag, if it is binary. Note that this cannot be used in the same SimpleTag as TagString.</documentation>
     <extension type="webmproject.org" webm="1"/>
   </element>
+  <element name="EditionFlagHidden" path="\Segment\Chapters\EditionEntry\EditionFlagHidden" id="0x45BD" type="uinteger" range="0-1" default="0" minOccurs="1" maxOccurs="1">
```

Comment:
Why is this section added to the end?
Wouldn't it be better to add this to the chapter section?

Reply:
I can do that